### PR TITLE
Fix check in/out computer and intercom manuals

### LINKED ISSUE #4277: Library search just a bit borked
Trying to search by category is a bit weird, if you search for the 'any' category, you won't find anything at all. If you try to use the category search on the actual checkout computer, it will open the window again after you select whatever category you want, and I'm pretty sure there's no way to get it back to going to any category

### LINKED ISSUE #8514: library issues
1- if you search books in the check in/out computer or the visitor computer and you choose a category, the option window appears 2 times, the first time with 6 options (any, fiction, non-fiction, adult, reference, religion), the second with 5 options only (all except "any"); the choose on the second window is ignored.

2- if you choose the "any" category it won't show any book, because the game thinks that "any" is a category itself; this means that if you choose a category you can't see the full list anymore.

3- the 'Ace Reporter Intercom manual' paper in the private study on boxstation have some outdated info (on a side note, the 'DJ Listening Outpost' paper on the dj satellite also has some wrong info).

stuff

### PR DESCRIPTION
**Pull request by xpcybic**
bugfix: "Fixed the check in/out computer prompting twice when filtering by category. Fixed selecting 'all' not actually working."
spellcheck: "Corrected the radio frequencies on the DJ satellite and ace reporter intercom manuals"
Also makes the ace reporter intercom manual a subtype instead of an instance. Finally, removes varediting on all the DJ/intercom manuals so they actually reflect this fix.
Fixes #8514
Fixes #4277

### CODE REVIEW

**Comment by gbasood** [comment]
100% tested right?

File: code/modules/library/computers/checkout.dm [hunk 1]
```diff
@@ -241,9 +241,10 @@
 	if(href_list["setcategory"])
 		var/newcategory = input("Choose a category to search for:") in (list("Any") + library_section_names)
 		if(newcategory)
```
Comment (clusterfack) [inline code/modules/library/computers/checkout.dm]: This line should be removed and the follow code should be tabbed back once

**Comment by clusterfack** [comment]
Otherwise looks good

File: code/modules/library/computers/checkout.dm (hunk 1 above)
Comment (clusterfack) [inline code/modules/library/computers/checkout.dm]: Unless you want to put + null in the options list to give it a cancel button

**Comment by xpcybic** [comment]
There's a second prompt for both category and author and they're for the new book upload
HOL UP while I fix this some more

**Comment by xpcybic** [comment]
Juan hundred percent tested™

**Pull request merged**